modules/core/shared/src/main/scala/lucuma/core/model/sequence/SequenceDigest.scala

```diff
@@ -21,25 +21,28 @@ import scala.collection.immutable.SortedSet
  *
  * @param observeClass   ObserveClass of the sequence as a whole
  * @param plannedTime    expected execution time for the sequence
- * @param offsets        set of offsets and guide statesthat are expected over the
+ * @param configs        set of offsets and guide states that are expected over the
  *                       course of the sequence execution
  * @param atomCount      number of atoms in the sequence
  * @param executionState completion state for this sequence
  */
 case class SequenceDigest(
   observeClass:   ObserveClass,
   timeEstimate:   CategorizedTime,
-  configs:        SortedSet[TelescopeConfig],
+  configs:        List[TelescopeConfig],
```

Comment:
As a `List`, there will be a config per step and a lot of duplicates which probably isn't what we want. I think we should keep just the unique configs for the digest.  If a customer wants the offset at each step they can get that in the full sequence.